scripts/generate_test_code.py

```diff
@@ -407,6 +412,8 @@ def validate_dependency(dependency):
     :return: input dependency stripped of leading & trailing white spaces.
     """
     dependency = dependency.strip()
+    if re.match(INVALID_NUMBER_FORMAT_REGEX, dependency, re.I):
+        raise GeneratorInputError('Invalid numerical format %s' % dependency)
```

Comment:
This only triggers if the dependency _as a whole_ starts with `0` and one or more digit. E.g. it matches

```
010
09
```

but not
```
-010
 010
FOO(010)
```

```suggestion
    m = re.search(r'\b0[0-9]+', dependency):
    if m:
        raise GeneratorInputError('Ambiguous integer literal: '+ m.group(0))
```